### PR TITLE
[ADD] purchase_order_add_variants

### PR DESCRIPTION
**Pull request by hugosantosred**
Module to quickly add variants from a template into a purchase order.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @rafaelbn to review

**Comment by liebana** [comment]
cc @gurneyalex He was also interested in the one we made for sales.

**Comment by liebana** [comment]
@rafaelbn @antespi It would be great if you could review too.

File: purchase_order_add_variants/README.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Contributors
+------------
+
+* Hugo Santos <hugo.santos@factorlibre.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
```
Comment (JayVora-SerpentCS) [inline purchase_order_add_variants/README.rst:34]: https

File: purchase_order_add_variants/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (JayVora-SerpentCS) [inline purchase_order_add_variants/README.rst:36]: https

File: purchase_order_add_variants/README.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline purchase_order_add_variants/README.rst:44]: https

File: purchase_order_add_variants/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#
+##############################################################################
+{
+    'name': 'Purchase Order Add Variants',
+    'summary': 'Add variants from template into purchase order',
+    'version': '1.0',
+    'author': 'FactorLibre,Odoo Community Association (OCA)',
+    'category': 'Purchase Management',
+    'license': 'AGPL-3',
+    'website': 'http://factorlibre.com',
```
Comment (JayVora-SerpentCS) [inline purchase_order_add_variants/__openerp__.py:28]: https://github.com/OCA/purchase-workflow

Comment (JayVora-SerpentCS) [reply purchase_order_add_variants/__openerp__.py:28]: Follow https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py for the conventions

File: purchase_order_add_variants/README.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Add variants in purchase orders
+===============================
+
+This module adds a wizard to select a template product showing all of this variants and allowing to change each variant quantity.
+Adding this variants to purchase order with the quantities selected
```
Comment (JayVora-SerpentCS) [inline purchase_order_add_variants/README.rst:8]: Can you explain why this feature would be needed ? Any use case plz.

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Pull request closed without merging**